resources/measures/ChangeBuildingLocation/tests/change_building_location_test.rb

```diff
@@ -38,6 +38,7 @@
 
 require 'openstudio'
 require 'openstudio/measure/ShowRunnerOutput'
+require_relative '../../../../test/helpers/minitest_helper'
 require 'fileutils'
```

Comment:
This file is missing `require 'minitest/autorun'`, which is in the other measure test files. I think since minitest_helper.rb contains `require 'minitest/autorun'` it should be fine.

Reply:
Yeah, I set all the test related library imported and configurations in the minitest_helper to keep it simper.